src/_pytest/helpconfig.py

```diff
@@ -212,6 +212,11 @@ def showhelp(config: Config) -> None:
     tw.line()
     tw.line("Environment variables:")
     vars = [
+        (
+            "CI",
+            "When truthy, pytest knows it is running in a CI process and does not truncate summary info",
```

Comment:
```suggestion
            "When set, regardless of the value, pytest knows it is running in a CI process and does not truncate summary info",
```

Reply:
good one :) thank you

I will not commit the suggestion directly as I also need to change the output of `pytest -h` in the doc/reference